<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>243</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>243</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>243</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>243</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>153</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>210 콤퓨타세탁 L</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>741</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="calendar">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Calendar</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>300</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="yearView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>210 콤퓨타세탁 L</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(143, 86, 59);</string>
      </property>
      <property name="text">
       <string>yyyy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QScrollArea" name="imageArea">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>561</width>
        <height>381</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>561</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="addPhoto">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>160</width>
        <height>81</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>251</red>
            <green>181</green>
            <blue>64</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>210 콤퓨타세탁 R</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(251, 181, 64);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="dateView">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>90</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>210 콤퓨타세탁 B</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(143, 86, 59);</string>
      </property>
      <property name="text">
       <string>dd</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="monthView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>70</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>210 콤퓨타세탁 R</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(143, 86, 59);</string>
      </property>
      <property name="text">
       <string>mm</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>620</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tag">
     <attribute name="title">
      <string>Map</string>
     </attribute>
     <widget class="QPushButton" name="tag_filter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>220</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="tag6_bt">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>a옛날목욕탕L</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-position: center center; 
	background-color:rgba(255,255,255,0);
	border-image:url(&quot;C:/Users/ilene/Documents/Visual Studio 2017/Qt Projects/CalendarFunction/tag6.png&quot;);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>181</width>
        <height>180</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tag1_bt">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>191</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>a옛날목욕탕L</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-position: center center; 
	background-color:rgba(255,255,255,0);
	border-image:url(&quot;C:/Users/ilene/Documents/Visual Studio 2017/Qt Projects/CalendarFunction/tag1.png&quot;);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>220</width>
        <height>210</height>
       </size>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tag4_bt">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>171</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>a옛날목욕탕L</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-position: center center; 
	background-color:rgba(255,255,255,0);
	border-image:url(&quot;C:/Users/ilene/Documents/Visual Studio 2017/Qt Projects/CalendarFunction/tag4.png&quot;);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>161</width>
        <height>160</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tag3_bt">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>151</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>a옛날목욕탕L</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-position: center center; 
	background-color:rgba(255,255,255,0);
	border-image:url(&quot;C:/Users/ilene/Documents/Visual Studio 2017/Qt Projects/CalendarFunction/tag3.png&quot;);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>190</width>
        <height>195</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="all_images">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>220</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
     <widget class="QToolButton" name="tag7_bt">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>430</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>a옛날목욕탕L</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-position: center center; 
	background-color:rgba(255,255,255,0);
	border-image:url(&quot;C:/Users/ilene/Documents/Visual Studio 2017/Qt Projects/CalendarFunction/tag7.png&quot;);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>231</width>
        <height>180</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tag8_bt">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>370</y>
        <width>171</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>a옛날목욕탕L</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-position: center center; 
	background-color:rgba(255,255,255,0);
	border-image:url(&quot;C:/Users/ilene/Documents/Visual Studio 2017/Qt Projects/CalendarFunction/tag8.png&quot;);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>161</width>
        <height>135</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tag2_bt">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>181</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>a옛날목욕탕L</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-position: center center; 
	background-color:rgba(255,255,255,0);
	border-image:url(&quot;C:/Users/ilene/Documents/Visual Studio 2017/Qt Projects/CalendarFunction/tag2.png&quot;);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>201</width>
        <height>210</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="tag5_bt">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>231</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>a옛날목욕탕L</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
	background-position: center center; 
	background-color:rgba(255,255,255,0);
	border-image:url(&quot;C:/Users/ilene/Documents/Visual Studio 2017/Qt Projects/CalendarFunction/tag5.png&quot;);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>241</width>
        <height>190</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextOnly</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
